From 21dfdf9224e856ee61cb0c3d8b38b3b568bf6b37 Mon Sep 17 00:00:00 2001
From: apporc <appleorchard2000@gmail.com>
Date: Mon, 6 Jul 2015 16:36:29 +0800
Subject: [PATCH 12/30] Updated requirements from stable/juno

Updated requirements from stable/juno, or the unittest will fail.
This is because many packages(oslo.*, eg) have largely changed
and are not working with juno.
Packages are set an upper limit for their version.

Signed-off-by: apporc <appleorchard2000@gmail.com>
---
 requirements.txt      | 76 +++++++++++++++++++++++++--------------------------
 test-requirements.txt | 28 +++++++++----------
 2 files changed, 52 insertions(+), 52 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index 2205362..882feb7 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -3,45 +3,45 @@
 # process, which may cause wedges in the gate later.
 
 pbr>=0.6,!=0.7,<1.0
-SQLAlchemy>=0.8.4,<=0.8.99,>=0.9.7,<=0.9.99
-anyjson>=0.3.3
+SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
+anyjson>=0.3.3,<=0.3.3
 argparse
-boto>=2.32.1
-decorator>=3.4.0
-eventlet>=0.15.1
-Jinja2
-keystonemiddleware>=1.0.0
-kombu>=2.5.0
-lxml>=2.3
-Routes>=1.12.3,!=2.0
-WebOb>=1.2.3
-greenlet>=0.3.2
-PasteDeploy>=1.5.0
-Paste
-sqlalchemy-migrate>=0.9.1,!=0.9.2
-netaddr>=0.7.12
-suds>=0.4
-paramiko>=1.13.0
-posix_ipc
-pyasn1
-Babel>=1.3
-iso8601>=0.1.9
+boto>=2.32.1,<2.35.0
+decorator>=3.4.0,<=3.4.0
+eventlet>=0.15.1,<=0.15.2
+Jinja2<=2.7.2
+keystonemiddleware>=1.0.0,<1.4.0
+kombu>=2.5.0,<=3.0.7
+lxml>=2.3,<=3.3.3
+Routes>=1.12.3,!=2.0,<=2.1
+WebOb>=1.2.3,<=1.3.1
+greenlet>=0.3.2,<=0.4.2
+PasteDeploy>=1.5.0,<=1.5.2
+Paste<=1.7.5.1
+sqlalchemy-migrate==0.9.1
+netaddr>=0.7.12,<=0.7.13
+suds==0.4
+paramiko>=1.13.0,<=1.15.2
+posix_ipc<=0.9.9
+pyasn1<=0.1.7
+Babel>=1.3,<=1.3
+iso8601>=0.1.9,<=0.1.10
 jsonschema>=2.0.0,<3.0.0
-python-cinderclient>=1.1.0
-python-neutronclient>=2.3.6,<3
-python-glanceclient>=0.14.0
-python-keystoneclient>=0.10.0
-six>=1.7.0
-stevedore>=1.0.0  # Apache-2.0
+python-cinderclient>=1.1.0,<=1.1.1
+python-neutronclient>=2.3.6,<2.4.0
+python-glanceclient>=0.14.0,<=0.15.0
+python-keystoneclient>=0.10.0,<1.2.0
+six>=1.7.0,<=1.9.0
+stevedore>=1.0.0,<=1.3.0 # Apache-2.0
 websockify>=0.6.0,<0.7
 wsgiref>=0.1.2
-oslo.config>=1.4.0  # Apache-2.0
-oslo.db>=1.0.0  # Apache-2.0
-oslo.rootwrap>=1.3.0
-pycadf>=0.6.0
-oslo.messaging>=1.4.0
-oslo.i18n>=1.0.0  # Apache-2.0
-lockfile>=0.8
-simplejson>=2.2.0
-rfc3986>=0.2.0  # Apache-2.0
-oslo.vmware>=0.6.0                      # Apache-2.0
+oslo.config>=1.4.0,<=1.6.0 # Apache-2.0
+oslo.db>=1.0.0,<1.1  # Apache-2.0
+oslo.rootwrap>=1.3.0,<=1.5.0
+pycadf>=0.6.0,<0.7.0  # Apache-2.0
+oslo.messaging>=1.4.0,<1.5.0
+oslo.i18n>=1.0.0,<=1.3.1 # Apache-2.0
+lockfile>=0.8,<=0.8
+simplejson>=2.2.0,<=3.3.1
+rfc3986>=0.2.0,<=0.2.0 # Apache-2.0
+oslo.vmware>=0.6.0,<0.9.0               # Apache-2.0
diff --git a/test-requirements.txt b/test-requirements.txt
index 519088a..afa013d 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -3,20 +3,20 @@
 # process, which may cause wedges in the gate later.
 
 hacking>=0.9.2,<0.10
-coverage>=3.6
-discover
-feedparser
-fixtures>=0.3.14
+coverage>=3.6,<=3.7.1
+discover<=0.4.0
+feedparser<=5.1.3
+fixtures>=0.3.14,<=1.0.0
 libvirt-python>=1.2.5                   # LGPLv2+
-mock>=1.0
-mox>=0.5.3
-MySQL-python
-psycopg2
+mock>=1.0,<=1.0.1
+mox>=0.5.3,<=0.5.3
+MySQL-python<=1.2.3
+psycopg2<=2.6
 pylint==0.25.2
-python-ironicclient>=0.2.1
-python-subunit>=0.0.18
+python-ironicclient>=0.2.1,<=0.3.3
+python-subunit>=0.0.18,<=1.0.0
 sphinx>=1.1.2,!=1.2.0,<1.3
-oslosphinx>=2.2.0  # Apache-2.0
-oslotest>=1.1.0  # Apache-2.0
-testrepository>=0.0.18
-testtools>=0.9.34
+oslosphinx>=2.2.0,<2.5.0  # Apache-2.0
+oslotest>=1.1.0,<1.4.0  # Apache-2.0
+testrepository>=0.0.18,<=0.0.20
+testtools>=0.9.34,!=1.4.0,<=1.5.0
-- 
2.4.3

